test/src/unit-regression1.cpp

```diff
@@ -1413,24 +1411,14 @@ TEST_CASE("regression tests 1")
         auto p1 = R"([{"op": "move",
                        "from": "/one/two/three",
                        "path": "/a/b/c"}])"_json;
-        CHECK_THROWS_AS(model.patch(p1), json::out_of_range&);
+        CHECK_THROWS_WITH_AS(model.patch(p1),
+                             "[json.exception.out_of_range.403] key 'a' not found", json::out_of_range&);
 
-        auto p2 = R"([{"op": "move",
```

Comment:
Why did you remove the test for `move` here?

Reply:
Not really removed. The test was executed twice: once to assert an exception is thrown, and second time to assert the exception's message. The new assertion macro does both checks at once, so I removed the redundant code.

Reply:
I had the exact same question, and then I realized that my confusion was mostly due to the way that the diff was aligned.  :)

Reply:
Yes, the diff aligning is a bit unfortunate in this case. The situation is more clear if you inspect the changes by commits, though.